src/imu_and_wheel_odom_node.cpp

```diff
@@ -195,11 +199,25 @@ int main(int argc, char **argv) {
         ROS_WARN("Parameter mag_north_correction_yaw is not a double, setting default 0 radians");
     }
 
-
-    // Evaluate alignment quternion
-    imu_alignment_.setRPY(imu_alignment_rpy_[0],
-                          imu_alignment_rpy_[1],
-                          imu_alignment_rpy_[2]);
+	// Get IMU->base_link tf
+	tf::TransformListener tf_listener;
```

Comment:
Pass `pn` to the constructor. This way, parameter and topic remappings are easier transferred also to the TF listener.
```suggestion
	tf::TransformListener tf_listener(pn);
```